README.md

```diff
@@ -59,6 +59,8 @@ projectlaunch.setup({
     split_focus_on_open = false,
     -- the filename of configuration file which under root directory.
     config_path = ".projectlaunch.json",
+    -- reload config after loaded a session or updated custom config
```

Comment:
Minor grammar correction, I think this should be:
```lua
-- automatically reload config after loading a session or updating your ProjectLaunch config file
```

Reply:
Sorry for my poor english, your comment is better than mine! 👍